### ISSUE TITLE
No matching version for @rescripts/rescript-use-eslint-config dependency in @rescripts/rescript-env

### ISSUE THREAD

**Issue by D4uS1** [rescripts/rescripts#129]
Hi everyone,

In the package.json file for @rescripts/rescript-env you are referencing to a non existing version of [@rescripts/rescript-use-eslint-config](https://github.com/harrysolovay/rescripts/blob/dd913e5e90a41b6afd711a4df8e06d0b1ab8a3cf/packages/rescripts/env/package.json#L13).

This version is not yet published to npm, hence it causes errors if i want to install the latest version of @rescripts/rescript-env.
The latest version published to npm is 0.0.11.

**Comment by urakozz** [comment]
True, in the repository it is actually updated https://github.com/harrysolovay/rescripts/blob/master/packages/rescripts/use-eslint-config/package.json#L3

It seems like it just have to be published

**Comment by urakozz** [comment]
@harrysolovay Hey, how do you think, when would it be possible to publish **rescript-use-eslint-config** ? :smiling_face_with_three_hearts:

**Comment by tannerhallman** [comment]
Several months later, same issue.. can I help to get this shipped?

**Comment by squidjam** [comment]
I am also experiencing this issue... Could we get a fix please?

**Comment by rodrix** [comment]
Same here:

My package.json dependencies looks like:

"devDependencies": {
		"@rescripts/cli": "0.0.16",
		"@rescripts/rescript-env": "0.0.14",
		...
}


Some fix?

**Comment by urakozz** [comment]
I decided “fuck this” and moved to craco

**Comment by squidjam** [comment]
I moved up to node 14 today for another reason and then updated the libs, and it's working for me now. 
Maybe the missing dependency has been uploaded/accepted at npm in the meantime.

**Comment by tannerhallman** [comment]
Sad.. but I switched to craco.
